### PR TITLE
prelaunch adjustments

### PR DESCRIPTION
**Pull request by heaversm**
- new hashed waitlist
- rework the requireRemovalUser middleware to first check the session variable, and if it is not present, check the hashed waitlist
- update start times of pilot, with a temporary date for the first that allows us to test before launch
- rework oauth step so that the removalPilot check and the session variable is set appropriately for either new or existing users
- set kanary session variable to false when user removes their monitor account if present
- set the optout to true on opt out now that we're done testing that bit
- update terms and privacy start dates for pilot

TODO before launch: adjust the first pilot start date before launch and remove unnecessary test accounts from waitlist

### CODE REVIEW

File: controllers/user.js [hunk 1]
```diff
@@ -787,6 +788,7 @@ async function handleRemovalOptout(req, res) {
   }
 
   req.session.kanary.onRemovalPilotList = false; //this must be set so they no longer see the tab in the navigation
+  req.session.destroy();
```
Comment (toufali) [inline controllers/user.js:791]: is line 790 setting var false redundant if we're just destroying the session next line?

File: removal-constants.js [hunk 2]
```diff
@@ -53,13 +52,14 @@ const REMOVAL_CONSTANTS = {
     {
       id: 1,
       name: "group_01",
-      start_time: 1641916800, //wed, jan 11 8am PST
+      start_time: 1642197405, //MH TODO: remove after testing
```
Comment (toufali) [inline removal-constants.js:55]: Does this need to be removed?

**Review by toufali — APPROVED** [review]

File: controllers/oauth.js [hunk 3]
```diff
@@ -101,6 +98,14 @@ async function confirmed(req, res, next, client = FxAOAuthClient) {
       fxaProfileData
     );
 
+    console.log("new subscriber", verifiedSubscriber);
```
[the remark this answers is not in the document]
Comment (heaversm) [reply controllers/oauth.js:101]: Nope - good catch. Will remove.

File: controllers/user.js (hunk 1 above)
Comment (heaversm) [reply controllers/user.js:791]: Hah...good point. I think so. Will remove.

File: removal-constants.js (hunk 2 above)
Comment (heaversm) [reply removal-constants.js:55]: I kept it in so QA could test - will change now for launch.

**Pull request merged**